src/utils/lockfile-manager.ts

```diff
@@ -0,0 +1,71 @@
+// Imports
+import tmp from 'tmp';
+import fs from 'fs';
+import path from 'path';
+
+/** Helper functions to preserve a lockfile */
+let tempDirectory: string | null = null;
+
+const lockfileManager = {
+	storeLockfile: (absolutePath: string): void => {
```

Comment:
If you rename the object/file/etc. to just “lockfile,” then you can just call this `store`.